integreat_cms/cms/templates/pois/poi_list_archived_row.html

```diff
@@ -3,11 +3,13 @@
 {% load content_filters %}
 <tr class="border-t border-gray-200 hover:bg-gray-100 text-gray-800">
     <td class="py-3 pl-4 pr-2">
-        <input type="checkbox"
-               name="selected_ids[]"
-               value="{{ poi.id }}"
-               form="bulk-action-form"
-               class="bulk-select-item" />
+        {% if perms.cms.change_poi %}
+            <input type="checkbox"
+                   name="selected_ids[]"
+                   value="{{ poi.id }}"
+                   form="bulk-action-form"
+                   class="bulk-select-item" />
+        {% endif %}
     </td>
     <td class="pr-2">
```

Comment:
```suggestion
    <td class="pr-2 {% if not perms.cms.change_poi %} py-3 pl-4 pr-2 {% endif %}">
```

Reply:
If you chose to go with this suggestion, please don't forget to also apply this to the `poi_list_archived.html` template :)

Reply:
the problem was with the row... so i did not change here anything :)